clang/test/Sema/riscv-vector-with-openmp.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -triple riscv64 -target-feature +v \
+// RUN:   -disable-O0-optnone -o - -fsyntax-only -fopenmp %s -verify 
```

Comment:
-disable-O0-optnone is pointless unless you're generating code *and* piping it to opt, but you're doing neither. Similarly -o - is pointless with -fsyntax-only -verify given you're not producing output (except from -verify itself directly to stdio).